tephra-core/src/main/java/org/apache/tephra/txprune/TransactionPruningRunnable.java

```diff
@@ -57,8 +60,13 @@ public void run() {
       Transaction tx = txManager.startShort();
       txManager.abort(tx);
 
+      if (tx.getInvalids().length == 0) {
+        LOG.info("Invalid list is empty, not running transaction pruning");
```

Comment:
so should we still prune here? It won't remove any invalid tx, but it will cleanup the prune state.